src/routes/api/editor/suggestions/[id]/dismiss/+server.ts

```diff
@@ -0,0 +1,18 @@
+import { error } from '@sveltejs/kit';
+
+export async function DELETE({ locals: { supabase, getSession }, params }) {
+	const session = await getSession();
+	if (!session) {
+		error(401, 'Unauthorized');
+	}
+
+	const suggestionId = params.id;
+
+	const { error: err } = await supabase.from('suggestions').delete().eq('id', suggestionId);
```

Comment:
Can we add amplitude tracking for this action? Would be nice to know how many and which suggestions are being dismissed.